### PR TITLE
Fix typo in "write_diagnostics_on_restart"

### PR DESCRIPTION
**Pull request by archermarx**
Really minor fix here. `write_diagnostics_on_restart` was `write_diagonstics_on_restart`.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Compiler settings
```
Comment (dpgrote) [inline install.sh]: Presumably this file was not supposed to be included in the PR.

Comment (archermarx) [reply install.sh]: Yes! apologies I thought i left it out

**Comment by dpgrote** [comment]
Thanks for the PR. Once the checks finish, I can approve and merge.

**Review by dpgrote — APPROVED** [review]
Thanks!

**Pull request merged**